examples/pennylane_run_qaoa_maxcut.py

```diff
@@ -0,0 +1,288 @@
+"""
+.. _qaoa_maxcut:
+
+Quantum Approximate Optimization for MaxCut
+===========================================
+
+In this tutorial, we implement a quantum approximate
+optimization algorithm (QAOA) for the MaxCut problem in Pennylane. (See `Farhi,
+Goldstone, and Gutmann (2014) <https://arxiv.org/abs/1411.4028>`__.) This
+example demonstrates how to sample joint qubit measurements from a variational
+quantum circuit to solve a combinatorial optimization problem.
```

Comment:
I'm not sure if this works 100% since the second sentence is more general than the first; how about switching them around to: "In this tutorial, we demonstrate how to sample joint qubit measurements from a variational quantum circuit to solve a combinatorial optimization problem. In particular, we implement the quantum approximate optimization algorithm (QAOA) for the MaxCut problem as proposed by `Farhi, Goldstone, and Gutmann (2014) <https://arxiv.org/abs/1411.4028>`__"

Reply:
That reads better!